lib/validations/validations.js

```diff
@@ -326,6 +459,10 @@ const Validations = function(config) {
               reject(err);
 
             } else if (!resp) {
+
+              if (!verifySignature(validation)) {
+                reject('invalid signature')
+              }
```

Comment:
this needs a return unless you want to both reject and continue to save the validation